auth/equinixmetal.go

```diff
@@ -22,26 +22,26 @@ import (
 	"path/filepath"
 )
 
-const PacketConfigPath = ".config/packet.json"
+const EquinixMetalConfigPath = ".config/equinixmetal.json"
```

Comment:
I'm wondering if this won't break if someone had a config named `packet.json`. This value is used as a default value for `--packet-config-file` (now `--equinixmetal-config-file`). I haven't seen `--packet-config-file` being used in our Jenkins setup, so not sure we should care.

It is also used in `ore equinixmetal --config-file` too.

Reply:
I had a doubt for this one, at first I started to support both - but I dropped the idea since it's a default config... Not sure about what we should do.

Reply:
Eh, if something breaks then we can revert this one line.